flake8_variables_names/checker.py

```diff
@@ -67,13 +69,13 @@ def get_varname_errors(self, var_name: str, var_ast_node) -> List[Tuple[int, int
             errors.append((
                 var_ast_node.lineno,
                 var_ast_node.col_offset,
-                'VNE001 single letter variable names are not allowed',
+                'VNE001 single letter variable names like \'%s\' are not allowed' % (var_name,),
```

Comment:
Change quotes here to comply PEP8, please (https://www.python.org/dev/peps/pep-0008/#string-quotes)